### DISCUSSION TITLE
[BUG] today style modifier overrides selected

### DISCUSSION THREAD

**Discussion by justduy** [gpbl/react-day-picker#1134] [Support]
EXPECTED BEHAVIOUR
---
When selecting today, it should apply all selected styling from `modifiersStyles`

ACTUAL BEHAVIOUR
---
Today's color overrides the selected color

CODE SANDBOX
---
https://codesandbox.io/s/21kx15xqyp

**EDIT:** Another bug is also displayed in this same sandbox, the selected day won't dissapear when changing months if style modifier is used.

**Comment by akigugale** [comment]
Hey @justduy, for the other bug you can use. Did you find a solution for the actual bug of styles being overridden?


```
<DayPickerInputer
    dayPickerProps={{ modifiersStyles: { ...modifiersStyles } }}
/>
```

[https://codesandbox.io/s/react-day-picker-base-forked-ccln2?file=/index.js](https://codesandbox.io/s/react-day-picker-base-forked-ccln2?file=/index.js)
